src/Stryker.Core/Stryker.Core/Initialisation/CsharpProjectComponentsBuilder.cs

```diff
@@ -82,7 +82,7 @@ private CsharpFolderComposite FindProjectFilesUsingBuildalyzer(IAnalyzerResult a
             // Save cache in a singleton so we can use it in other parts of the project
             FolderCompositeCache<CsharpFolderComposite>.Instance.Cache = cache;
 
-            CSharpParseOptions cSharpParseOptions = BuildCsharpParseOptions(analyzerResult, options);
```

Comment:
Could you create a new issue for tracking removal of the BuildCsharpParseOptions method and replacing it with GetParseOptions?